dockers/docker-fpm-frr/bgpd.conf.default.j2

```diff
@@ -36,7 +37,53 @@ route-map ISOLATE permit 10
 route-map set-next-hop-global-v6 permit 10
   set ipv6 next-hop prefer-global
 !
+{% set vrf_lo_intf = {} %}
+{% for (name, prefix) in LOOPBACK_INTERFACE|pfx_filter %}
+{%   set intf_vrf = 'default' %}
+{%   if LOOPBACK_INTERFACE.has_key(name) and LOOPBACK_INTERFACE[name].has_key('vrf_name') %}
+{%     set intf_vrf = LOOPBACK_INTERFACE[name]['vrf_name'] %}
+{%   endif %}
+{%   if vrf_lo_intf.has_key(intf_vrf) or vrf_lo_intf.update({intf_vrf:[]}) %}
+{%   endif %}
+{%   if name not in vrf_lo_intf[intf_vrf] and vrf_lo_intf[intf_vrf].append(name) %}
+{%   endif %}
+{% endfor %}
+{% set vrf_routers = [] %}
+{% set vrf_bgp_nbr = {} %}
+{% for nbr_key in BGP_NEIGHBOR %}
```

Comment:
I suggest to put this into bgpd.peer template.

Reply:
This can not move out because peer-group PEER_V4/PEER_V6 need config in these vrfs.